Source/sounds.c

```diff
@@ -237,207 +237,6 @@ sfxinfo_t original_S_sfx[] = {
 
   // [crispy] play DSSECRET if available
   { "secret",  sg_none,  100, 0, -1, -1, 0 },
-
```

Comment:
I am a bit sad we are losing this build-time initialization here, but I guess it's fine to do it at run-time as well.